endpoints/v2/v2auth.py

```diff
@@ -133,7 +133,7 @@ def generate_registry_jwt(auth_result):
     token = generate_bearer_token(
         audience_param, subject, context, access, TOKEN_VALIDITY_LIFETIME_S, instance_keys
     )
-    return jsonify({"token": token})
+    return jsonify({"token": token.decode('ascii')})
```

Comment:
Are we 100% sure JWT tokens are ASCII compatible? We should check the spec and link to the relative section

Reply:
The tokens should be a base64 encoded string joined by a `.` AFAIK.

Reply:
The first line of the [abstract in the RFC](https://tools.ietf.org/html/rfc7519): 
> JSON Web Token (JWT) is a compact, **URL-safe** means...
Yeah it's just base 64 character set + periods.